### PR TITLE
CORE-8462: changed the request body and the email format of the feedb…

### PR DESCRIPTION
**Pull request by dennisroberts71**
…ack endpoint

### CODE REVIEW

File: src/terrain/util/email.clj [hunk 1]
```diff
@@ -81,30 +81,12 @@
...
-     :to        (config/feedback-dest-addr)
-     :from-addr (config/feedback-dest-addr)
-     :subject   "DE Feedback"
-     :template  "blank"
-     :values    {:contents text})))
+  [{:strs [email fields subject]}]
+  (send-email
+   :to        (config/feedback-dest-addr)
+   :from-addr (or email (:email current-user))
+   :subject   (or subject "DE Support Request")
```
Comment (psarando) [inline src/terrain/util/email.clj]: The default subject for this `send-feedback-email` function used to be `DE Feedback`. Why change it to `DE Support Request`?

File: src/terrain/util/email.clj [hunk 2]
```diff
@@ -81,30 +81,12 @@
...
+   :template  "blank"
+   :values    {:contents (apply str (mapv (fn [[k v]] (str k ": " v "\n")) fields))}))
```
Comment (psarando) [inline src/terrain/util/email.clj]: My initial instinct would be to use `clojure.string/join` here.

Maybe something like
```clojure
(string/join "\n" (mapv (partial string/join ": ") fields))
```

Of course, this current method works just as well :)

Comment (dennisroberts71) [inline src/terrain/util/email.clj]: Heh, that would work. I wanted to get the trailing newline in there, though. :)

Comment (psarando) [inline src/terrain/util/email.clj]: Ah, good point, I missed that 👍

File: src/terrain/util/email.clj (hunk 1 above)
Comment (dennisroberts71) [inline src/terrain/util/email.clj]: This is part of the move away from having us receive "feedback" any time a user encounters a job failure. I suppose I could change the URL path to reflect something more accurate...

Comment (psarando) [inline src/terrain/util/email.clj]: Ah OK, I knew we were adding user support emails, but I didn't realize we were getting rid of general feedback too.

**Comment by psarando** [comment]
Looks good. I'd say :shipit:, but I don't know if we want to wait for UI changes to handle the new endpoint?

**Comment by dennisroberts71** [comment]
Thanks for the review!

**Pull request merged**